apps/app/src/views/routes/Prescription.tsx

```diff
@@ -495,6 +512,68 @@ export const Prescription = () => {
                 </Text>
               )}
             </HStack>
+
+            {orders.length === 0 ? null : (
+              <>
+                <Divider />
+
+                <Text color="gray.500" fontWeight="medium" fontSize="sm">
+                  Orders
+                </Text>
+                <VStack spacing={4} w="full">
+                  {orders.map((fill: Maybe<Fill>) => {
+                    if (!fill) return null;
+                    const address = fill?.order?.pharmacy?.address;
+                    const addressString = address
+                      ? `${address?.street1}, ${address?.city}, ${address?.state} ${address?.postalCode}`
+                      : '';
+                    return (
+                      <LinkBox key={fill.id} w="full" style={{ textDecoration: 'none' }}>
+                        <Card
+                          variant="outline"
+                          p={[2, 3]}
+                          w="full"
+                          shadow="none"
+                          _hover={{
+                            backgroundColor: 'gray.50'
+                          }}
+                        >
+                          <HStack justifyContent="space-between">
+                            <VStack alignItems="start">
+                              <HStack>
+                                <LinkOverlay href={`/orders/${fill?.order?.id}`}>
+                                  <HStack spacing={2}>
+                                    <Text fontSize="md">
+                                      {fill?.order?.pharmacy?.name ||
+                                        'Pharmacy has not been selected by the patient.'}
```

Comment:
nit: think something like "Waiting for pharmacy selection" is better, no "." either

Reply:
Currently using "Pending selection" on orders table, maybe same for consistency?